cmd/kpod/docker/types.go

```diff
@@ -0,0 +1,271 @@
+package docker
```

Comment:
I think this file should go into containers/storage or containers/image, and then we would not need to cut and paste it.  @nalind WDYT?